src/model/CustomMenuHolder.php

```diff
@@ -89,6 +89,8 @@ public function getCMSFields()
     function requireDefaultRecords()
     {
         parent::requireDefaultRecords();
+        $config = SiteConfig::current_site_config();
+        $menus = CustomMenuHolder::get();
 
         // Main Menu
         if ($this->class == 'CustomMenuHolder') {
```

Comment:
I think this if statement needs looking at, it seems pretty out of date.

I don't think we need the `if ($this->class == 'CustomMenuHolder') {` wrapping the logic.

I think we can also remove the if statements with `Database::alteration_message("Main menu created", "created");` as I think this code is SS3 specific?